### PR TITLE
Make new homepage live

### PR DESCRIPTION
**Pull request by dwhenry**
Move experiement homepage to production.

### CODE REVIEW

File: app/views/home/show.html.erb [hunk 1]
```diff
@@ -1,126 +1,144 @@
...
+        </div>
+      </a>
+    </div>
 
-    <p>
-      <a href="/appointments" class="button">
-        Book a free appointment
+    <div class="l-home-options__option-container l-home-options__option-container--full-width">
+      <a class="l-home-options__link l-home-options__link--mix-options" href="/mix-options">
+        <div class="l-home-options__option l-home-options__option--mix-options"><span class="circle circle--s circle--mix-options l-home-options__circle"></span><span class="l-home-options__text">Mix your options</span>
```
Comment (benlovell) [inline app/views/home/show.html.erb]: Can this be wrapped like the other options?

File: app/views/home/show.html.erb [hunk 2]
```diff
@@ -1,126 +1,144 @@
...
+</div>
+
+<div class="l-grid-row l-home-row l-home-row--accent">
+  <div class="l-page-container">
+    <div class="l-column-two-thirds">
+      <h1 class="l-home-single-column-heading">Welcome to Pension Wise</h1>
+      <p>Free and impartial government guidance to help you understand what you can do with your pension pot.</p>
+    </div>
+    <div class="l-column-third l-home-intro-container">
+      <%= inline_svg "home/folks.svg", class: "l-home-folks" %>
```
Comment (benlovell) [inline app/views/home/show.html.erb]: Some double quoted strings have snuck in here and I feel it's my duty as human linter to inform you of this 😄 🤖

**Comment by dwhenry** [comment]
@benlovell - those are now fixed
@benbarnett - Added the test back in as discussed

**Comment by benlovell** [comment]
Don't forget to squash 😉

**Comment by dwhenry** [comment]
@benlovell had to merge that one bit of HTML onto a single line - added a comment to explain why. :)

**Comment by Guntrisoft** [comment]
@dwhenry still needs squashing before merge :)

**Pull request merged**